.bumpversion.cfg

```diff
@@ -17,3 +18,9 @@ values =
 
 [bumpversion:file:jupyterlab_celltests/_version.py]
 
+[bumpversion:file:package.json]
```

Comment:
Only makes sense if the js and python pkgs are to be released in lockstep...are they?